Cargo.toml

```diff
@@ -130,7 +130,7 @@ debug = false
 [profile.release]
 lto = true
 strip = true
-opt-level = 3
```

Comment:
Even if it's the default I don't see the value in not being explicit about it. This line should be kept.